src/convertToObject.js

```diff
@@ -12,7 +12,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  let array = sourceString.split(';');
+
+  const result = {};
+
+  array = array.map(item => (item.replace('"').trim()));
```

Comment:
```suggestion
  array = array.map(item => (item.trim()));
```

This 'replace' does nothing in this case.